Makefile

```diff
@@ -1,65 +1,66 @@
-# Copyright (C) 2017-2020  The SymbiFlow Authors.
```

Comment:
Reminder to myself -- Looks like I need to restore the make targets I seem to have accidentally removed.

Reply:
Fixed.